global.json

```diff
@@ -1,9 +1,11 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.2.20120.3"
+    "version": "5.0.100-preview.2.20120.3",
```

Comment:
```suggestion
    "version": "5.0.100-preview.2.20163.4",
```

Reply:
I don't want to update the necessary SDK as desktop msbuild should still work with 16.4. I just want to update the bootstrapped dotnet SDK.

Reply:
We tend to move these values in sync and expect devs to at least run `./restore` before using VS

Reply:
I think you will need to switch to MSBuild 16.5.  The updates to dotnet/sdk to support net5.0 as the TargetFramework [use new intrinsic MSBuild functions](https://github.com/dotnet/sdk/blob/983c8115e6341287a5866ed7be05c663c7e89614/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L72-L75) which were [added in MSBuild 16.5](https://github.com/microsoft/msbuild/pull/4911).

Reply:
Won't changing the required VS or MSBuild version to 16.5 in this file just cause Windows builds to fail faster on these agents? From the errors, they only have 16.4 installed.